extensions/amp-subscriptions/0.1/renderer.js

```diff
@@ -29,12 +30,17 @@ export class Renderer {
     /** @const @private */
     this.ampdoc_ = ampdoc;
 
+    this.document_ = ampdoc.win.document;
```

Comment:
If all the same, I'd rather not store a document reference here, but instead inline it where it's used - it's a dangerous API somewhat.